packages/ui-extensions/src/surfaces/checkout/api/address-autocomplete/standard.ts

```diff
@@ -193,6 +193,8 @@ interface Extension<
    * * [`collect_buyer_consent.sms_marketing`](https://shopify.dev/docs/api/checkout-ui-extensions/configuration#collect-buyer-consent): the extension can collect customer consent for SMS marketing.
    *
    * * [`collect_buyer_consent.customer_privacy`](https://shopify.dev/docs/api/checkout-ui-extensions/configuration#collect-buyer-consent): the extension can register customer consent decisions that will be honored on Shopify-managed services.
+   *
+   * * [`iframe.sources`](https://shopify.dev/docs/api/checkout-ui-extensions/configuration#iframe): the extension can embed an external URL in an iframe.
```

Comment:
```suggestion
   * * iframe.sources: the extension can embed an external URL in an iframe.
```
Nit: The docs won't be updated to include the `iframe` capabilities with this update. @jffortier should we remove the link for now?